apps/commudle-admin/src/app/feature-modules/events/components/event-form-responses/event-form-responses.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component, OnInit, TemplateRef, ViewChild } from '@angular/core';
 import { FormBuilder, FormGroup } from '@angular/forms';
 import { ActivatedRoute } from '@angular/router';
 import { ColumnMode, SortType } from '@commudle/ngx-datatable';
-import { NbWindowService } from '@commudle/theme';
+import { NbDialogService, NbWindowService } from '@commudle/theme';
```

Comment:
if NbWindowService not in use, lets remove this

Reply:
NbWindowService is also being used